src/mol-plugin/headless-plugin-context.ts

```diff
@@ -1,25 +1,29 @@
 /**
- * Copyright (c) 2019-2022 mol* contributors, licensed under MIT, See LICENSE file for more info.
+ * Copyright (c) 2019-2023 mol* contributors, licensed under MIT, See LICENSE file for more info.
```

Comment:
date should be when file was created and when last change was made, so just 2023